topics/reactiveui/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Paul Betts
+display_name: ReactiveUI
+github_url: https://github.com/reactiveui/reactiveui
+logo: reactiveui.png
```

Comment:
Did you mean to include an image file? If not, this line should be removed.

Reply:
https://camo.githubusercontent.com/0e6578a14a6bd4654449f88ee838f77700d6d070/68747470733a2f2f692e696d6775722e636f6d2f32336b666253392e706e67

Reply:
Ah, can you add that file as `reactiveui.png` in this branch? It should be in the topics/reactiveui/ directory. It will also need to be 288x288.

Reply:
Ooops, my bad! Already fixed that.